deployment/debian-package-armhf/Dockerfile.amd64

```diff
@@ -0,0 +1,42 @@
+FROM debian:9
```

Comment:
Why not use the official dotnet stretch image instead? https://github.com/dotnet/dotnet-docker/blob/master/TAGS.md#linux-amd64-tags

Reply:
I'm really not a fan of that image, brcause it doesn't install the package for dotnet just throws the binaries in random folders. I haven't bothered wiwith the other one because it works, but I like the consistency of the packages.